test/image/mocks/bar_stackto100_negative.json

```diff
@@ -1,9 +1,33 @@
 {
     "data":[
-        {"name":"Col1","y":["1","2","3","4","5"],"x":["1","2","3","4","5"],"type":"bar","uid":"aeb9ea"},
-        {"name":"Col2","y":["2","3","4","3","2"],"x":["1","2","3","4","5"],"type":"bar","uid":"2f201d"},
-        {"name":"Col3","y":["5","4","3","2","1"],"x":["1","2","3","4","5"],"type":"bar","uid":"aef0bf"},
-        {"name":"Col4","y":["-1","0","1","0","-1"],"x":["1","2","3","4","5"],"type":"bar","uid":"330b4d"}
+        {
+            "name":"Col1",
+            "y":["1","2","3","4","5"],
+            "x":["1","2","3","4","5"],
+            "type":"bar",
+            "uid":"aeb9ea"
+        },
+        {
+            "name":"Col2",
+            "y":["2","3","4","3","2"],
+            "x":["1","2","3","4","5"],
+            "type":"bar",
+            "uid":"2f201d"
+        },
+        {
+            "name":"Col3",
+            "y":["5","4","3","2","1"],
+            "x":["1","2","3","4","5"],
+            "type":"bar",
+            "uid":"aef0bf"
```

Comment:
Sorry to keep adding more things that need changes but could you remove the `uid` here as well too please!